docs/user_guides/Gufi for Non Administrative users/main.tex

```diff
@@ -0,0 +1,177 @@
+\documentclass{article}
+
+% Language setting
+% Replace `english' with e.g. `spanish' to change the document language
+\usepackage[english]{babel}
+
+% Set page size and margins
+% Replace `letterpaper' with`a4paper' for UK/EU standard size
+\usepackage[letterpaper,top=2cm,bottom=2cm,left=3cm,right=3cm,marginparwidth=1.75cm]{geometry}
+
+% Useful packages
+\usepackage{amsmath}
+\usepackage{graphicx}
+\usepackage[colorlinks=true, allcolors=blue]{hyperref}
+
+\title{GUFI: A Common User's Guide}
```

Comment:
No need for `Common`

Reply:
Same with the directory name - no need for `Non Administrative`

Reply:
No more Non Administrative directory due to modularization.

File Names are now:
gufi_administrator.tex
gufi_developer.tex
gufi_user.tex